src/Instagram-Basic/Provider.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
Sorry, just noticed the folder name does not comply with PSR-4. Could you rename it to `InstagramBasic`? (will add a CI step for this as well)

Reply:
Done